test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -233,7 +257,11 @@ export default class PerfCompare extends Component {
 
         let baselineRunJSON = new JenkinsRunJSON(resBenchmarkBaselineJson);
         let testRunJSON = new JenkinsRunJSON(resBenchmarkTestJson);
-        
+        await this.getChildrenList(resBenchmarkBaselineJson);
+        let childListBaselineJsonMetrics = this.state.childList;
```

Comment:
Can't we just directly put the result from `getChildrenList` into `childListBaselineJsonMetrics` instead of using `this.state.childList`?